go/client/logclient_test.go

```diff
@@ -275,7 +275,7 @@ func TestGetProofByHash(t *testing.T) {
 	}
 
 	for _, tc := range tests {
-		resp, err := c.GetProofByHash(tc.hash, tc.treesize)
+		resp, err := c.GetProofByHash(context.TODO(), tc.hash, tc.treesize)
```

Comment:
I guess that in a test, it should be `context.Background()`? Not like we'll refactor in the future to get the context from somewhere else, right?